src/twisted/conch/test/test_keys.py

```diff
@@ -188,6 +193,21 @@ def test__guessStringType(self):
         )
         self.assertEqual(keys.Key._guessStringType(b"not a key"), None)
 
+    def test_OpenSSH_cert_not_supported(self):
+        """
+        OpenSSH certificates are not yet supported.
+        """
+        self.assertRaises(
+            keys.BadKeyError,
+            keys.Key._guessStringType,
```

Comment:
I don't think that is a good idea to write the test for the private method.

Why not use the public API `fromString` ?